tests/git_server/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
Is there any reason to make this file owned by root with the 755 rights? It's uncomfortable to work with.

Reply:
I don't understand what you mean. Git doesn't record ownership or file permissions (beyond executable/non-executable); this is just a normal executable file.

Reply:
I get the following problem: when I change the branch, the file is missing and shown as removed in git, and I get the following error:

```
$ git checkout tests/git_server/entrypoint.sh
error: unable to create file tests/git_server/entrypoint.sh: Permission denied
Updated 1 path from the index
```